src/components/TabsItem/TabsItem.tsx

```diff
@@ -67,6 +73,22 @@ export const TabsItem = ({
       );
   }
 
+  if (process.env.NODE_ENV === "development" && isTabFlow) {
+    if (!restProps["aria-controls"]) {
+      warn(`Передайте в "aria-controls" id контролируемого блока`, "warn");
+    } else if (!restProps["id"]) {
+      warn(
+        `Передайте "id" компоненту для использования в "aria-labelledby" контролируемого блока`,
+        "warn"
+      );
+    }
+  }
+
+  let tabIndex: React.HTMLAttributes<HTMLElement>["tabIndex"] = undefined;
+  if (isTabFlow) {
+    tabIndex = selected ? 0 : -1;
+  }
```

Comment:
```jsx
const resolveTabIndex = (selected, tabIndexDefault = 0) => selected ? tabIndexDefault : -1;

// ...

export const TabsItem = ({
  // ...
  tabIndex: tabIndexProp,
  ...restProps
}: TabsItemProps) => { /* ... */ }
```

```suggestion
  const tabIndex: React.HTMLAttributes<HTMLElement>["tabIndex"] = isTabFlow ? resolveTabIndex(selected, disabled, tabIndexProp) : tabIndexProp;;
```

`tabIndex` может придти извне, возможно сохранить обратную совместимость. Что думаешь?

Reply:
Не уверен, что имеет смысл уносить это в отдельную функцию, а то, что может прийти отдельно надо тоже обработать, согласен

Reply:
Я предполагал, что `tabIndex` надо только при `selected` выставлять, а иначе всегда выставлять `-1` 🤔 

> PS: в отдельную функцию вынес, чтобы не городить много условий, в контексте того, что выше написал

В данный момент вот такой сценарий:

1. По клавише **TAB** мы фокусируемся на **выбранный** `TabsItem` внутри `Tabs`
2. Клавишами **ARROW**, **HOME** и **END** навигируемся между соседними `TabsItem` в пределах `Tabs`
3. При повторном нажатии **TAB** или **SHIFT** + **TAB** мы выходим за пределы `Tabs`.

Если передим кастомный `tabIndex`, то **п.1** и **п.3** будут работать иначе.

Вот пример:
```jsx
<Tabs>
  <TabsItem id="one" tabIndex={0}></<TabsItem>
  <TabsItem id="two" selected></<TabsItem>
</Tabs>
```

- По **п.1** мы будем фокусироваться в первую очередь на `id="one"`, т.к. он имеет `tabIndex` выше `-1`.
- По **п.3** клавиша **TAB** и **SHIFT** + **TAB** будут навигировать нас между `id="one"` и `id="two"`.

---

Не знаю как правильней) Оставлять такую возможность или нет 🤔

Reply:
Ну ладно, думаю это не такая уж и большая проблема, очень пограничный кейс